src/compose/ComposeBox.js

```diff
@@ -217,6 +219,36 @@ class ComposeBox extends PureComponent<Props, State> {
     }
   };
 
+  replaceMessageTextAtRange = (text: string, beg: number, end: number) => {
```

Comment:
Abbreviations like "beg" can be confusing and we use them sparingly. A good name for this would be "start".

Reply:
there is no need for this now, in this PR, as I have removed this function in newer implementation, I will keep this in mind though for future contributions.